producers/http/handlers_test.go

```diff
@@ -161,6 +161,27 @@ func TestContainersHandler(t *testing.T) {
 			So(resp.StatusCode, ShouldEqual, http.StatusOK)
 			So(strings.TrimSpace(string(got)), ShouldEqual, strings.TrimSpace(string(expected)))
 		})
+		Convey("Should return container IDs when app metrics, but not container metrics, are present", func() {
+			port := setup([]producers.MetricsMessage{testNodeData, testAppData})
+			resp, err := http.Get(urlBuilder("localhost", port, "/v0/containers"))
+			if err != nil {
+				panic(err)
+			}
+			defer resp.Body.Close()
+
+			got, err := ioutil.ReadAll(resp.Body)
```

Comment:
I think I'd prefer this test to readlike:

```
var containerIDs []string
if err := json.NewDecoder(resp.Body).Decode(&containerIDs); err != nil {
    panic(err)
}
So(resp.StatusCode, ShouldEqual, http.StatusOK)
So(containerIDs, ShouldResemble, []string{testContainerData.Dimensions.ContainerID})
```

Assuming that ShouldResemble is something like `reflect.DeepEquals`

Not a huge deal but that would be more readable in my opinion.

Reply:
I was following the conventions in the test file, but this is way neater.